R/lists_subscribers.R

```diff
@@ -42,64 +32,28 @@ lists_subscribers <- function(list_id = NULL,
                               parse = TRUE,
                               token = NULL) {
 
-  if (n > 5000) {
-    n <- ceiling(n / 5000)
-    count <- 5000
-  } else {
-    count <- n
-    n <- 1L
-  }
-  args <- list(
+  params <- lists_params(
     list_id = list_id,
     slug = slug,
-    owner_user = owner_user,
-    n = count,
-    cursor = cursor,
-    parse = parse,
-    token = token
+    owner_user = owner_user
   )
-  out <- vector('list', n)
-  for (i in seq_along(out)) {
-    out[[i]] <- do.call("lists_subscribers_call", args)
-    args$cursor <- attr(out[[i]], "next_cursor")
-    if (is.null(args$cursor) || identical(args$cursor, "0")) {
-      break
-    }
-  }
-  if (parse) {
-    out <- do_call_rbind(out)
-  }
-  out
-}
-
 
-lists_subscribers_call <- function(list_id = NULL,
-                                   slug = NULL,
-                                   owner_user = NULL,
-                                   n = 20,
-                                   cursor = "-1",
-                                   parse = TRUE,
-                                   token = NULL) {
-  
-  params <- lists_params(
-    list_id = list_id,
-    slug = slug,
-    owner_user = owner_user,
-    count = n,
-    cursor = cursor
+  r <- TWIT_paginate_cursor(token, "/1.1/lists/subscribers", params, 
+    n = n,
+    cursor = cursor, 
+    page_size = 200,
```

Comment:
Again, the page_size is less than the maximum allowed (5000) on the documentation. 
```suggestion
    page_size = 5000,
```